consensus/service/src/config/tokens.rs

```diff
@@ -11,61 +11,63 @@ use mc_transaction_core::{tokens::Mob, Token, TokenId};
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::{fs, iter::FromIterator, path::Path};
 
-mod der_signer_set {
+mod pem_signer_set {
     use super::*;
+    use pem::Pem;
 
-    /// A helper struct for ser/derserializing a SignerSet that uses hex-encoded
-    /// DER representation of Ed25519 public keys.
+    /// A helper struct for ser/derserializing an Ed25519 SignerSet that is PEM
+    /// encoded.
     #[derive(Serialize, Deserialize)]
-    struct DerSignerSet {
-        signers: Vec<String>,
+    struct PemSignerSet {
+        signers: String,
         threshold: u32,
     }
 
-    /// Helper method for serializing a SignerSet<Ed25519Public> into a
-    /// hex-DER-encoded variant.
+    /// Helper method for serializing a SignerSet<Ed25519Public> into PEM.
     pub fn serialize<S: Serializer>(
         signer_set: &Option<SignerSet<Ed25519Public>>,
         serializer: S,
     ) -> Result<S::Ok, S::Error> {
-        let der_signer_set = signer_set.as_ref().map(|signer_set| DerSignerSet {
-            signers: signer_set
+        let pem_signer_set = signer_set.as_ref().map(|signer_set| {
+            let pems = signer_set
                 .signers()
                 .iter()
-                .map(|signer| hex::encode(&signer.to_der()))
-                .collect(),
-            threshold: signer_set.threshold(),
+                .map(|signer| Pem {
+                    tag: String::from("FOO"),
```

Comment:
How about "signer"?
```suggestion
                    tag: "signer".to_string(),
```

Reply:
oops, I missed that. it should be `PUBLIC KEY` to match what openssl generates, but in general has no meaningful implication.